src/components/ContactForm/index.jsx

```diff
@@ -11,18 +55,20 @@ export default function ContactForm() {
     <StyledDiv>
       <h2>Candy Fluffs Updates!</h2>
       <p>Sign up to get updates about new products and when I’ll be at anime conventions!</p>
+      <div className="form">
       <form method="post" netlify-honeypot="bot-field" data-netlify="true" name="contact" >
-        <label htmlFor="firstName">First Name</label>
-        <input type="text" name="firstName"/>
-        <label htmlFor="lastName">Last Name</label>
-        <input type="text" name="lastName"/>
-        <label htmlFor="email">Email</label>
-        <input type="text" name="email"/>
+        {/* <label htmlFor="firstName">First Name</label> */}
+        <input type="text" name="firstName" value="First Name"/>
+        {/* <label htmlFor="lastName">Last Name</label> */}
+        <input type="text" name="lastName" value="Last Name"/>
+        {/* <label htmlFor="email">Email</label> */}
+        <input type="text" name="email" value="E-mail address"/>
```

Comment:
Overall it looks great, but can you change the "value" to "placeholder" (or whatever the syntax really is) so that the user can change the value?